app/helpers/katello/concerns/settings_helper_extensions.rb

```diff
@@ -45,6 +45,16 @@ def katello_template_setting_values(name)
         templates = ProvisioningTemplate.where(:template_kind => TemplateKind.where(:name => name))
         templates.each_with_object({}) { |tmpl, hash| hash[tmpl.name] = tmpl.name }.to_json
       end
+
+      def default_download_policy_setting_values
```

Comment:
This seems like a really strange name. What does default mean? I would have probably gone with something like `download_policy_values`.